actions/distribute_workers.py

```diff
@@ -45,6 +45,12 @@ def distribute_idle_workers(self):
                 mineral_field = self.mineral_fields.closest_to(drone)
                 local_controller.add_action(drone.gather(mineral_field))
 
+    def drone_dodge(self):
+        """Avoiding effects"""
+        for drone in self.ai.drones:
+            if self.dodge_effects(drone):
```

Comment:
the if is not required, since we do no further actions on the drone here.

Reply:
Ill remove it until we find a way to do it correctly